Models/RegistrationBase.php

```diff
@@ -359,12 +363,108 @@ public static function create($request)
         $reg->fill($inputs);
         $reg->save();
 
+
         $response['success'] = true;
         $response['data']['item'] = $reg;
         $response['messages'][] = trans('vaahcms-general.saved_successfully');
         return $response;
 
     }
+
+    //-------------------------------------------------
+
+    public static function createRegistration($request)
+    {
+
+        $inputs = $request->all();
+
+           $rules = [
+                'first_name' => 'required|max:150',
+                'last_name' => 'required|max:150',
+                'username' => 'required|max:150',
+               'email' => 'required|regex:/(.+)@(.+)\.(.+)/i',
```

Comment:
@we-anil-y001 i am not sure this is the correct `regex`, write the source of `regex`

Reply:
'email' => 'email:rfc,dns', I have use this and source code https://laravel.com/docs/10.x/validation